entrypoint.sh

```diff
@@ -62,6 +64,11 @@ esac
 
 configure_runner() {
   ARGS=()
+  if [[ -n "${APP_ID}" ]] && [[ -n "${APP_PRIVATE_KEY}" ]]; then
+    echo "Obtaining access token for app_id ${APP_ID}"
+    ACCESS_TOKEN=$(APP_ID="${APP_ID}" APP_PRIVATE_KEY="${APP_PRIVATE_KEY}" bash /app_token.sh)
+  fi
+
```

Comment:
You mean if it fails to get the ACCESS_TOKEN? Otherwise we actually need to fall through. Using APP_ID and APP_PRIVATE_KEY we can acquire an ACCESS_TOKEN, with this ACCESS_TOKEN, we subsequently acquire a RUNNER_TOKEN.